Jenkinsfile

```diff
@@ -69,30 +112,37 @@ pipeline {
                                 retry(3) {
                                     checkout scm
                                 }
-                                withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings') {
+                                withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '-Xmx8192M -Xss128M -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC -Djava.security.egd=file:/dev/./urandom') {
                                     withCredentials([string(credentialsId: 'sonarqube-token', variable: 'SONAR_TOKEN')]) {
                                         sh 'mvn -q -B -Dfindbugs.skip=true -Dcheckstyle.skip=true org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar -Dsonar.host.url=https://sonarqube.com -Dsonar.login=$SONAR_TOKEN  -Dsonar.organization=codice -Dsonar.projectKey=ddf -pl !$DOCS,!$ITESTS'
                                     }
                                 }
                             }
                         },
+                        // Coverity will be skipped on all PR builds
                         coverity: {
                             node('linux-medium') {
-                                retry(3) {
+                              script {
+                                if (env.CHANGE_ID != null) {
+                                  echo "Skippng Coverity on PR build"
```

Comment:
anyway to 'return' here so we don't need to nest the rest of this in an else statement?

Reply:
A green checkmark still shows up on the PR builds, that might not be a problem, but may confuse people as if the PR had no major coverity issues.

Reply:
Yeah, I think theres not much we can do currently to avoid a green check mark. Though I'm curious if we will be able to have conditional parallel steps when the declarative pipelines fully support parallel stages (Should  be in the next release of the pipeline-model-definition)